pybamm/input/parameters/lithium-ion/cathodes/LFP_Prada2013/README.md

```diff
@@ -0,0 +1,7 @@
+# Lithium Cobalt Oxide cathode parameters
```

Comment:
If the 2017 references are added to complete the set it should be mentioned here